packages/x-data-grid-premium/src/hooks/features/pivoting/useGridPivoting.tsx

```diff
@@ -459,6 +480,19 @@ export const useGridPivoting = (
     [apiRef, computePivotingState],
   );
 
+  const sidebarPreProcessing = React.useCallback<GridPipeProcessor<'sidebar'>>(
```

Comment:
Following the conventions in the rest of code base?
```suggestion
  const addPivotingPanel = React.useCallback<GridPipeProcessor<'sidebar'>>(
```